common/lib/xmodule/xmodule/video_module/video_utils.py

```diff
@@ -57,6 +57,21 @@ def rewrite_video_url(cdn_base_url, original_video_url):
         return None
 
     parsed = urlparse(original_video_url)
+
+    # malformed input, say not scheme, results in the netloc
+    # being returned as an empty string.
+    if not parsed.netloc:
+        return None
+
+    # Only re-write URLs if the original netloc is in the whitelist, otherwise
+    # no guarantees can be made that the re-write target can service the request.
+    try:
+        if parsed.netloc not in whitelist:
+            return None
+    except TypeError:
+        log.exception("message")
+        return None
```

Comment:
Is the long term approach here to log as a warning? Or just swallow it entirely?